src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -281,7 +285,6 @@ private function denormalizeObject($data, $class, $format, array $context = arra
         foreach ($this->normalizers as $normalizer) {
             if ($normalizer instanceof DenormalizerInterface
                 && $normalizer->supportsDenormalization($data, $class, $format)) {
-
                 return $normalizer->denormalize($data, $class, $format, $context);
```

Comment:
This align the `&&` with the `return`, reducing readability. Maybe the if should be put on one line.